client/src/Reception/Reception.js

```diff
@@ -130,15 +131,22 @@ export class ReceptionView {
   initCallbacks() {
     const _this = this;
 
-    this.joinButton.onclick = function () {
-      _this.dispose();
-      const menuAuth = new MenuAuthView(_this.webSocketService);
-      document.body.appendChild(menuAuth.html());
-      menuAuth.setOnClose(function () {
-        menuAuth.dispose();
-        document.body.appendChild(_this.html());
-      });
-    };
+    SystemUtils.File.loadJSON('../assets/config/config_features.json').then(
```

Comment:
config should be loaded in bootstrap.js then pass as parameter to ReceptionView

Reply:
Oh yes it is actually quite simple 😓

Reply:
Change in this commit https://github.com/VCityTeam/UD-Imuv/pull/135/commits/a931e6557bc6590bf5fabbaf16bb55d53b8593cf